docs/0.2/creating_schemas.md

```diff
@@ -232,7 +232,8 @@ identified in the schema definition.
 
    You don't have to use the same file that was used to create the schema,
    but the file must specify the same name and present the properties in the
-   same order.
+   same order. Only new properties may be added. Existing properties may not be
```

Comment:
"may not" is this enforced?

Reply:
you can do it but the transaction will fail